pyxnat/core/derivatives/freesurfer.py

```diff
@@ -89,24 +89,44 @@ def aparc(self, atlas='desikan-killiany'):
 
         res2 = [e for e in res if e.startswith('# Measure ')]
         for each in volumes:
-            m = float([e for e in res2
-                       if each in e.split(', ')[1]][0].split(', ')[-2])
-            table.append([None, 'Volume_mm3', each, m])
+            m = [e for e in res2 if each == e.split(', ')[1]]
+            if len(m) > 1:
+                msg = 'Key %s is ambiguous in file %s (found entries: %s)'\
+                      % (each, f._uri, m)
+                raise Exception(msg)
```

Comment:
Could we better use a more specific Exception type such as `ValueError`?

Reply:
Fine. Should be all in in dfeb78b.